modules/library/http/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- =======================================================================    
+        Maven Project Configuration File                                        
+                                                                                
+        The Geotools Project                                                    
+            http://www.geotools.org/                                            
+                                                                                
+        Version: $Id$              
+     ======================================================================= -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>library</artifactId>
+        <groupId>org.geotools</groupId>
+        <version>25-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>gt-http</artifactId>
```

Comment:
Just reviewing now, a shame we did not have this to discuss at our bi-weekly meeting as it is an interesting topic.

I think if we adopt this functionality it would end up going into a core library such as metadata or referencing (rather than introducing new jars / dependencies).

Reply:
It was discussed in the open already, bug tracker and list, and we suggested to move it to a separate module, to avoid adding a new dependency (commons http client) to all geotools modules.

Reply:
I feel that this functionality has been somewhat hidden away in Geotools. And as #2804 suggest, there are more usages of http-calls than I've handled.
I think it would've been great if all http traffic is handled likewise throughout Geotools. Then it would be easier to deal with bug's regarding https / proxy, and add functionality like logging and url-controlling (ref. #2804 ).
Whether that would be in a new namespace within gt-main, moved to a different library as gt-referencing or gt-metadata (of which I would prefer gt-metadata), or as a separate module in gt-http isn't up to me to decide.
It's perfectly fine for me to wait for the next bi-weekly meeting.